synapse/lib/stormtypes.py

```diff
@@ -8068,7 +8068,9 @@ class User(Prim):
          'type': {'type': 'function', '_funcname': '_methUserAddRule',
                   'args': (
                       {'name': 'rule', 'type': 'list', 'desc': 'The rule tuple to add to the User.', },
-                      {'name': 'gateiden', 'type': 'str', 'desc': 'The gate iden used for the rule.', 'default': None, }
+                      {'name': 'gateiden', 'type': 'str', 'desc': 'The gate iden used for the rule.',
+                       'default': None, },
+                      {'name': 'indx', 'type': 'int', 'desc': 'The index to add the rule in.', 'default': None, }
```

Comment:
desc is a bit awkwardly phrased here and below. I like the desc from above. Perhaps something like `The position of the rule as a 0 based index.`